### PR TITLE
feat: add useStore

### PR DESCRIPTION
**Pull request by yqrashawn**
- Please approve this after you fully understand what's going on.
- Please do point out places that might be wrong/misleading.
- Please give some suggestions if there are 
	- better documentation
	- better implementation

This change is [Reviewable](https://reviewable.io/reviews/conflux-chain/helios/81)

### CODE REVIEW

**Review by angelia-yuqi-personal — APPROVED** [review]

File: packages/popup/src/store/useStore.js [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+        isValidating: newIsValidating,
+        set,
+        get,
+      })
+
+    return {
+      data: newData,
+      error: newError,
+      isValidating: newIsValidating,
+      mutate,
```
Comment (angelia-yuqi-personal) [inline packages/popup/src/store/useStore.js:83]: it may be newMutate？

File: packages/popup/src/store/useStore.js [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+
+  return store
+})
+
+export const useStore = (args = {}) => {
+  const store = useGlobalStore()
+
+  const storeProxy = new Proxy(store, {
+    get() {
+      const key = arguments[1]
```
Comment (angelia-yuqi-personal) [inline packages/popup/src/store/useStore.js:122]: why the second args is key here?

**Review by angelia-yuqi-personal — CHANGES_REQUESTED** [review]

File: packages/popup/src/store/useStore.js (hunk 2 above)
Comment (yqrashawn) [reply packages/popup/src/store/useStore.js:122]: it's by design, here's the doc https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Proxy

Comment (angelia-yuqi-personal) [reply packages/popup/src/store/useStore.js:122]: get it～

**Review by angelia-yuqi-personal — APPROVED** [review]

File: packages/popup/src/store/rpc-config.js [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+    //
+    // const {get, set, deps, params, opts} = args
+    //
+    // get/set is the fn from zustand store
+    //
+    // deps/params/opts is the params to be called by swr
+    /// deps by default will be the rpc method ("wallet_getAccountGroup" here)
+    /// deps will be null when calling useStore with deps: null
+    /// deps will be a array when calling deps with any other arguments
+    /// deps is undefined by default
```
Comment (angelia-yuqi-personal) [inline packages/popup/src/store/rpc-config.js:37]: it is conflict with line 34，according code，it should be deleted.

**Review by angelia-yuqi-personal — CHANGES_REQUESTED** [review]

**Review by angelia-yuqi-personal — DISMISSED** [review]

**Review by angelia-yuqi-personal — APPROVED** [review]

**Pull request merged**